sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -1339,13 +1353,29 @@ public void importBatch(FinishBundleContext context) throws IOException {
       this.fhirStore = fhirStore;
     }
 
-    /**
-     * Instantiates a new Execute bundles.
-     *
-     * @param fhirStore the fhir store
-     */
-    ExecuteBundles(String fhirStore) {
-      this.fhirStore = StaticValueProvider.of(fhirStore);
+    @Override
+    ValueProvider<String> getFhirStore() {
+      return null;
```

Comment:
shouldn't you return the fhirStore here?

Reply:
Done